{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/cloudsim-legacy.json"}, "html": {"href": "#!/osrf/cloudsim-legacy"}, "avatar": {"href": "data/bytebucket.org/ravatar/{e799e7f6-0caa-464a-97f8-8605a37ebfb1}ts=python"}}, "type": "repository", "name": "cloudsim-legacy", "full_name": "osrf/cloudsim-legacy", "uuid": "{e799e7f6-0caa-464a-97f8-8605a37ebfb1}"}, "links": {"attachments": {"href": "data/repositories/osrf/cloudsim-legacy/issues/36/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/cloudsim-legacy/issues/36.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/cloudsim-legacy/issues/36/watch"}, "comments": {"href": "data/repositories/osrf/cloudsim-legacy/issues/36/comments_page=1.json"}, "html": {"href": "#!/osrf/cloudsim-legacy/issues/36/gzclient-causes-crashes-of-master-might-be"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/cloudsim-legacy/issues/36/vote"}}, "reporter": {"display_name": "michele hallak", "uuid": "{f8cca72b-2f8a-4372-bcb5-10ea801a3fe9}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bf8cca72b-2f8a-4372-bcb5-10ea801a3fe9%7D"}, "html": {"href": "https://bitbucket.org/%7Bf8cca72b-2f8a-4372-bcb5-10ea801a3fe9%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/d10108e88396c34c2d474c28bba19974d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsMH-1.png"}}, "nickname": "mhallak", "type": "user", "account_id": "557058:643e25c4-f0cd-4e12-8c67-7a038a7e7e40"}, "title": "gzclient causes crashes of master - Might be issue unrelated to cloudsim", "component": null, "votes": 0, "watches": 1, "content": {"raw": "I don't know if this bug is related to cloudsim.\r\n\r\nRun configuration trio.\r\nRun atlas simulation (launched via console).\r\neverything is ok with simulator computer, field computer and local vpn connected computer: run rviz ok and rzgraph also.\r\n\r\nAs soon as running gzclient, everything crashes:\r\n\r\nthe field computer lost connection to master, the local VPN computer as well, so I guess that the master has crashed on the simulator computer.\r\n\r\nError [InsertModelWidget.cc:252] model.config file is missing in directory[\"/usr/local/share/drcsim-1.3/models/test\"]\r\nQt has caught an exception thrown from an event handler. Throwing\r\nexceptions from an event handler is not supported in Qt. You must\r\nreimplement QApplication::notify() and catch all exceptions there.\r\n\r\nterminate called after throwing an instance of 'Ogre::ItemIdentityException'\r\n  what():  OGRE EXCEPTION(5:ItemIdentityException): Can not find codec for 'png' image format.\r\nThere are no formats supported (no codecs registered). in Codec::getCodec at /build/buildd/ogre-1.7.4/OgreMain/src/OgreCodec.cpp (line 69)\r\n", "markup": "markdown", "html": "<p>I don't know if this bug is related to cloudsim.</p>\n<p>Run configuration trio.\nRun atlas simulation (launched via console).\neverything is ok with simulator computer, field computer and local vpn connected computer: run rviz ok and rzgraph also.</p>\n<p>As soon as running gzclient, everything crashes:</p>\n<p>the field computer lost connection to master, the local VPN computer as well, so I guess that the master has crashed on the simulator computer.</p>\n<p>Error [InsertModelWidget.cc:252] model.config file is missing in directory[\"/usr/local/share/drcsim-1.3/models/test\"]\nQt has caught an exception thrown from an event handler. Throwing\nexceptions from an event handler is not supported in Qt. You must\nreimplement QApplication::notify() and catch all exceptions there.</p>\n<p>terminate called after throwing an instance of 'Ogre::ItemIdentityException'\n  what():  OGRE EXCEPTION(5:ItemIdentityException): Can not find codec for 'png' image format.\nThere are no formats supported (no codecs registered). in Codec::getCodec at /build/buildd/ogre-1.7.4/OgreMain/src/OgreCodec.cpp (line 69)</p>", "type": "rendered"}, "assignee": null, "state": "resolved", "version": null, "edited_on": null, "created_on": "2013-02-12T09:55:52.822676+00:00", "milestone": null, "updated_on": "2013-02-12T11:03:16.011582+00:00", "type": "issue", "id": 36}